lib/modulesync.rb

```diff
@@ -183,9 +184,14 @@ def self.update(options)
     exit 1 if errors && options[:fail_on_warnings]
   end
 
-  def self.pr(module_options = {})
-    github_conf = module_options[:github]
-    gitlab_conf = module_options[:gitlab]
+  def self.pr(module_options)
+    if !module_options.nil?
```

Comment:
Would the code be easier if we used `module_options ||= {}`

Reply:
This is pretty much what I had in the signature before.

As a reminder, the reason for this was to support modules without options (when `managed_modules.yaml` is a list, not a hash).

Reply:
The signature alone didn't fix the issue (as demonstrated by the tests I added). The problem is, when you set the method's parameter to `nil` explicitly the default value doesn't help. The change should now lead to the same result, if I see this correctly.

As for the syntactic sugar, I'm not a Ruby person, so feel free to direct me to what is a shorter, equivalent way of writing the code.

Reply:
I applied the syntactic sugar. Looks much nicer now, thanks! :+1: